modules/ztp-preparing-the-hub-cluster-for-ztp.adoc

```diff
@@ -47,6 +47,19 @@ $ oc patch argocd openshift-gitops \
 -n openshift-gitops --type=merge \ 
 --patch-file out/argocd/deployment/argocd-openshift-gitops-patch.json
 ----
++
+[NOTE]
+====
+For a disconnected environment, you must patch the `out/argocd/deployment/argocd-openshift-gitops-patch.json` file with the `ztp-site-generator` image mirrored in your local registry by running the following command:
```

Comment:
```suggestion
For a disconnected environment, patch the `out/argocd/deployment/argocd-openshift-gitops-patch.json` file with the `ztp-site-generator` image that you mirrored in your local registry. Run the following command:
```

Reply:
Also it's `ztp-site-generate` not `ztp-site-generator`

Reply:
done. Thanks.